recipe/meta.yaml

```diff
@@ -27,6 +27,10 @@ source:
       # with the usual relocation treatement and leads to other issues, see #935;
       # replace with placeholder that allows interested users to fix the file path
       - patches/0003-don-t-bake-non-relocatable-CMAKE_INSTALL_FULL_LIBDIR.patch
+      # apparently, someone changed the artifacts under
+      #   * https://github.com/substrait-io/substrait/archive/v0.6.0.tar.gz
+      #   * https://apache.jfrog.io/artifactory/arrow/thirdparty/7.0.0/mimalloc-v2.0.6.tar.gz
+      - patches/0004-fix-changed-thirdparty-artifact-hashes.patch
```

Comment:
@kou @kkraus14 @xhochy 

Do we need to be concerned that the hashes for the versioned tarballs of substrait and mimalloc changed? This happened sometime in the last couple of hours - I assume it's benign, but in the absolute worst case it could indicate a compromised artifact.

@conda-forge/arrow-cpp

Reply:
I'm guessing this is related to https://github.com/community/community/discussions/45830